test/unit/assignments.spec.ts

```diff
@@ -1,13 +1,23 @@
 import * as util from "../util";
 
 test("const declaration", () => {
-    const lua = util.transpileString(`const foo = true;`);
-    expect(lua).toBe(`local foo = true`);
+    const lua = util.testFunction`const foo = true;`.getMainLuaCodeChunk();
+    expect(lua).toContain(`local foo = true`);
 });
 
 test("let declaration", () => {
-    const lua = util.transpileString(`let foo = true;`);
-    expect(lua).toBe(`local foo = true`);
+    const lua = util.testFunction`let foo = true;`.getMainLuaCodeChunk();
+    expect(lua).toContain(`local foo = true`);
+});
```

Comment:
```ts
util.testModule`
	${kind} foo = true;
	// @ts-ignore
	return (globalThis as any).foo;
`.expectToEqual(undefined)
```

Reply:
In case you've missed it, it actually wasn't resolved

Reply:
I left these tests but also added that test below, as I feel they actually test different things.

Reply:
Well, the test you have added below is indeed different, but the one I've written above should be functionally equivalent

Reply:
They are not equivalent. The current test asserts a variable statement with local is created, the other test just asserts there is no variable with that name in the global scope. That test would also pass if we ignore the declaration completely, or if we export it under some other name